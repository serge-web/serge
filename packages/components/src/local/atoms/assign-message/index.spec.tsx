/* global it expect */

import React from 'react'
import renderer from 'react-test-renderer'

import AssignMessage from './index'

describe('DropdownList component:', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <AssignMessage
          options={['Game Control', 'CO', 'Blue 1']}
        />
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
