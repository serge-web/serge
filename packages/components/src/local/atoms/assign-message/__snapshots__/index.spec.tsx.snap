// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DropdownList component: renders correctly 1`] = `
<div
  className="MuiFormControl-root makeStyles-margin-1"
>
  <div
    className="MuiInputBase-root MuiInput-root makeStyles-dropdownInput-2 MuiInputBase-formControl MuiInput-formControl"
    onClick={[Function]}
  >
    <select
      aria-invalid={false}
      className="MuiNativeSelect-root MuiNativeSelect-select MuiInputBase-input MuiInput-input"
      disabled={false}
      onAnimationStart={[Function]}
      onBlur={[Function]}
      onChange={[Function]}
      onFocus={[Function]}
      required={false}
      value="1"
    >
      <option
        disabled={true}
        selected={true}
        value="0"
      >
        Select Item
      </option>
      <option
        value="1"
      >
        Item 1
      </option>
      <option
        value="2"
      >
        Item 2
      </option>
      <option
        value="3"
      >
        Item 3
      </option>
    </select>
    <svg
      aria-hidden={true}
      className="MuiSvgIcon-root MuiNativeSelect-icon"
      focusable="false"
      viewBox="0 0 24 24"
    >
      <path
        d="M7 10l5 5 5-5z"
      />
    </svg>
  </div>
</div>
`;
